modules/test/playwright/tests/frontend-data-set-views-web/settings.spec.ts

```diff
@@ -0,0 +1,163 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {featureFlagsTest} from '../../fixtures/featureFlagsTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {dataSetManagerApiHelpersTest} from './fixtures/dataSetManagerApiHelpersTest';
+import {dataSetsPageTest} from './fixtures/dataSetsPageTest';
+import {fieldsPageTest} from './fixtures/fieldsPageTest';
+import {settingsPageTest} from './fixtures/settingsPageTest';
+import {visualizationModesPageTest} from './fixtures/visualizationModesPageTest';
+import {DEFAULT_LABEL} from './utils/constants';
+
+export const test = mergeTests(
+	dataSetsPageTest,
+	dataSetManagerApiHelpersTest,
+	featureFlagsTest({
+		'LPD-10735': true,
+		'LPS-164563': true,
+		'LPS-178052': true,
+	}),
+	fieldsPageTest,
+	loginTest(),
+	settingsPageTest,
+	visualizationModesPageTest
+);
+
+test.beforeEach(async ({dataSetManagerApiHelpers}) => {
+	await dataSetManagerApiHelpers.createDataSet({erc: 'SettingsDataSetERC'});
+	await dataSetManagerApiHelpers.createDataSetView({
+		r_fdsEntryFDSViewRelationship_c_fdsEntryERC: 'SettingsDataSetERC',
+	});
```

Comment:
As per [playwright guidelines](https://liferay.atlassian.net/wiki/spaces/ENGR/pages/2729705766/Writing+tests+with+Playwright#Randomizing-names), we should use `getRandomString()`s in cases like this. This will ensure that junk from one failed test will not affect following ones.